jest.config.base.js

```diff
@@ -38,4 +38,8 @@ module.exports = {
 
     // An array of regexp patterns that are matched against all source file paths before re-running tests in watch mode
     watchPathIgnorePatterns: ['libDev', 'lib'],
+
+    // An array of regexp pattern strings that are matched against all module paths before those paths are
+    // to be considered 'visible' to the module loader
+    modulePathIgnorePatterns: ['libDev'],
```

Comment:
is that intentional change? libDev update was triggering tests or something like that?

Reply:
Intentional. Jest was trying to load two colliding mocks (from `src/__mocks__/*` and  `/libDev/__mocks__/*`). I hope it doesn't break anything, but it feels like it shouldn't